### PR TITLE
add io.gitlab.irrlamb

### PR DESCRIPTION
**Pull request by jazztickets**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

This will replace io.github.jazztickets.irrlamb

### CODE REVIEW

**Comment by jazztickets** [comment]
bot, build io.gitlab.irrlamb

File: io.gitlab.irrlamb.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+app-id: io.gitlab.irrlamb
```
Comment (hfiguiere) [inline io.gitlab.irrlamb.yml:1]:
```suggestion
app-id: io.gitlab.jazztickets.irrlamb
```

Excactly like with github

File: io.gitlab.irrlamb.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
```
Comment (hfiguiere) [inline io.gitlab.irrlamb.yml:3]:
```suggestion
runtime-version: '22.08'
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: io.gitlab.irrlamb.yml (hunk 1 above)
Comment (jazztickets) [reply io.gitlab.irrlamb.yml:1]: The new webpage is https://irrlamb.gitlab.io and https://gitlab.com/irrlamb. Shorter is better.

File: io.gitlab.irrlamb.yml (hunk 2 above)
Comment (jazztickets) [reply io.gitlab.irrlamb.yml:3]: I tried 22.08 and it causes all sorts of problems with irrlicht. I can't use it at this time.

File: io.gitlab.irrlamb.yml (hunk 1 above)
Comment (hfiguiere) [reply io.gitlab.irrlamb.yml:1]: but you don't own `gitlab.io`, so an acceptable app-id would be `io.gitlab.irrlamb.irrlamb`

Comment (jazztickets) [reply io.gitlab.irrlamb.yml:1]: Seems pretty silly to me, but whatever i'll just patch it in to be jazztickets

Comment (hfiguiere) [reply io.gitlab.irrlamb.yml:1]: it's not silly. It's even in the documentation.

**Pull request closed without merging**